### PR TITLE
perf: use system's own flag on non-Windows systems

### PR DESCRIPTION
**Pull request by kecrily**
This is an addition to https://github.com/eslint/new.eslint.org/pull/213

### CODE REVIEW

File: src/assets/scss/tokens/typography.scss [hunk 1]
```diff
@@ -78,3 +77,7 @@
             "Segoe UI Emoji",
             "Segoe UI Symbol";
 }
+
+.win {
+    font-family: "Twemoji Country Flags";
```
Comment (nzakas) [inline src/assets/scss/tokens/typography.scss:82]: Do we really want to set this as the font family for the whole page?

Comment (kecrily) [reply src/assets/scss/tokens/typography.scss]: Now I've changed it to use only in the language selector

File: src/assets/scss/tokens/typography.scss [hunk 2]
```diff
@@ -79,5 +79,5 @@
 }
 
 .win {
-    font-family: "Twemoji Country Flags";
+    font-family: "Twemoji Country Flags"!important;
```
Comment (nzakas) [inline src/assets/scss/tokens/typography.scss:82]:
```suggestion
    font-family: "Twemoji Country Flags" !important;
```

**Review by nzakas — APPROVED** [review]

**Comment by nzakas** [comment]
Thanks!

**Pull request merged**